float32 in_path_vel
float32 rotate_vel
bool rotate_to_path
bool rotate_at_start
